packages/framework/src/evaluate.ts

```diff
@@ -1,10 +1,10 @@
-import { isEmpty, last, merge } from "lodash-es";
+import { isEmpty, isObjectLike, last, merge, toString } from "lodash-es";
 import type { InvokableMethods, ScreenContextDefinition } from "./state";
 import { EnsembleStorage } from "./storage";
 
 export const buildEvaluateFn = (
   screen: ScreenContextDefinition,
-  js?: string,
+  js?: unknown,
```

Comment:
This should remain a string, any formatting should happen in the invoking component